modules/openapi-generator/src/main/resources/python/api_client.mustache

```diff
@@ -372,7 +376,10 @@ class ApiClient:
             # and attributes which value is not None.
             # Convert attribute name to json key in
             # model definition for request.
-            obj_dict = obj.to_dict()
+            if hasattr(obj, 'to_dict') and callable(getattr(obj, 'to_dict')):
+                obj_dict = obj.to_dict()
+            else:
+                obj_dict = obj.__dict__
```

Comment:
looks like this is fixing another issue, right?

is there a minimal spec to reproduce the issue? sorry I might have missed it